content/docs/how-we-work/task-estimation/estimation-process.md

```diff
@@ -0,0 +1,35 @@
+---
+
+# Estimation Process
+
+It’s essential to discuss the estimation process during onboarding to understand how it’s handled for each project. Different projects might have varying estimation methods. Below are some commonly encountered practices:
+
+## Grooming
+
+We ensure that the necessary grooming is done, and the required details are added to the Implementation details Jira field. The Technical Architect (TA) can be consulted if guidance on the approach is needed. Adding implementation details helps the client understand where the hours will be utilized.
+
+## Estimation Templates
+
+Use the [master estimate sheet template](https://docs.google.com/spreadsheets/d/1wXAFXsSbiQfaMSCMGr-7GvWHYh2O9fc5mhUOQgJsos8/edit?gid=683107236#gid=683107236) and follow the instructions given in the sheet. The team can raise a request or suggest feedback or invite a call to discuss any challenges, which will help revise the estimation sheet iteratively. Add estimates in internal and bug tickets as well and get those reviewed by the TA. Add a link and a screenshot of the estimation sheet on the respective ticket. The screenshot is important as it will help us identify any changes done later to the estimations. We also use the [CDM ticket estimation template](https://docs.google.com/spreadsheets/d/1NExM6FlLe0kWwhFNPp4RBi8E1D7iixRQHPVriP22B1Y/edit?gid=0#gid=0) only for ticket-level estimation.
+
+## Sprint Planning Meetings
+
+Estimation is often conducted during sprint planning meetings. This allows the team to discuss and refine estimates based on the latest information and priorities.
+
+## Planning Poker
+
+Planning poker is a gamified technique used to estimate sprints and tasks, engaging the entire team. This method is commonly applied in staffing projects with Acquia. The process involves the Technical Architect (TA), Project Manager (PM), or sprint planner explaining a task and its features. The team then estimates the story points required to complete the task, with each member voting on their estimate.
+
+The results are revealed, and team members with extreme estimates explain their reasoning. This discussion aims to uncover new approaches or overlooked aspects of the task. Senior developers or TAs often provide insights into the complexities involved, helping the team reach a consensus.
+
+Over consecutive sprints, a seasoned team improves its estimates as they gain a deeper understanding of the project. This exercise also reveals hidden scenarios and impacts, correcting both underestimates and overestimates. Tools like PlanITpoker are often used for online scrum planning poker.
+
+## Pre-Estimated by TA and PM
+
+In some staffed projects, though not recommended, the TA and PM may pre-estimate tasks, with the team only reviewing and discussing potential issues with these estimates.
```

Comment:
@Yemaneberhan-Lemma I think we can remove this as its specific to Staff aug. 

@swarad07 can include this in Axelerant Handbook specific to Staff Aug

Reply:
It is already mentioned that this is for staffed projects, no?

Reply:
@swarad07 @Yemaneberhan-Lemma Above was with assumption that we want to have the content generic and not specific to any Program.

However if @Yemaneberhan-Lemma and @hussainweb are ok with it then this is fine.